prime-router/metadata/fhir_mapping/hl7/datatypes/XTN/ContactPoint.yml

```diff
@@ -0,0 +1,335 @@
+
+value_2:
+  condition: $telEquipType NULL && $localNum NULL && $unformattedTelNum NULL && $telNum NOT_NULL
+  valueOf: $telNum
+  vars:
+    telNum: String, XTN.1
+    telEquipType: String, XTN.3
+    localNum: String, XTN.7
+    unformattedTelNum: String, XTN.12
+
+value_3:
+  condition: $telEquipType EQUALS Internet || $telEquipType EQUALS X.400
+  valueOf: $commAddress
+  vars:
+    telEquipType: String, XTN.3
+    commAddress: String, XTN.4
+
+value_4:
+  condition: $telEquipType NOT_EQUALS Internet && $telEquipType NOT_EQUALS X.400 && $unformattedTelNum NOT_NULL
+  valueOf: unformattedTelNum
+  vars:
+    telEquipType: String, XTN.3
+    unformattedTelNum: String, XTN.12
+
+value_5:
+  condition: $telEquipType NULL && $unformattedTelNum NOT_NULL
+  valueOf: unformattedTelNum
+  vars:
+    telEquipType: String, XTN.3
+    unformattedTelNum: String, XTN.12
+
+#mapping from HL7 Concept Map: TelecomUseCode
+#https://docs.google.com/spreadsheets/d/1EE3a5UIGXLi8wMDr49fWTEVx8QS2GFqYgitE3vLOQTQ/edit#gid=0
+use_1:
+  condition: $useCode EQUALS PRN
+  type: STRING
+  value: "home"
+  vars:
+    useCode: String, XTN.2
+
+use_2:
+  condition: $useCode EQUALS WPN
+  type: STRING
+  value: "work"
+  vars:
+    useCode: String, XTN.2
+
+use_3:
+  condition: $useCode EQUALS BPN
+  type: STRING
+  value: "mobile"
+  vars:
+    useCode: String, XTN.2
+
+#Element is not in XTN[ContactPoint] but rather is specified in PID[Patient] mapping inventory
+use_4:
+  condition: $useCode NULL
+  type: STRING
+  value: "home"
+  vars:
+    useCode: String, XTN.2
+
+#mapping from HL7 Concept Map: TelecomEquipmentType
+#https://docs.google.com/spreadsheets/d/1FhoR5oxEWdHmpR5S0_wYPstdy1lwv7rL-JWJu_DE_Vs/edit#gid=0
+system_1:
+  condition: $equipType EQUALS PH
+  type: STRING
+  valueOf: "phone"
+  vars:
+    equipType: String, XTN.3
+
+system_2:
+  condition: $equipType EQUALS FX
+  type: STRING
+  valueOf: "fax"
+  vars:
+    equipType: String, XTN.3
+
+system_3:
+  condition: $equipType EQUALS Internet || $equipType EQUALS X.400
+  type: STRING
+  valueOf: "email"
+  vars:
+    equipType: String, XTN.3
+
+system_4:
+  condition: $equipType EQUALS BP
+  type: STRING
+  valueOf: "pager"
+  vars:
+    equipType: String, XTN.3
+
+system_5:
+  condition: $equipType EQUALS MD || $equipType EQUALS SAT || $equipType EQUALS TDD || $equipType EQUALS TTY
+  type: STRING
+  valueOf: "other"
+  vars:
+    equipType: String, XTN.3
+
+system_6:
+  condition: $telEquipType NULL && $commAddress NOT_NULL
+  valueOf: "email"
+  vars:
+    telEquipType: String, XTN.3
+    commAddress: String, XTN.4
+
+system_7:
+  condition: $telEquipType NULL && $commAddress NULL
+  expressionType: nested
+  vars:
+    telEquipType: String, XTN.3
+    commAddress: String, XTN.4
+  expressionsMap:
+    extension:
+      generateList: true
+      expressionType: nested
+      expressionsMap:
+        url:
+          type: SYSTEM_URL
+          value: "data-absent-reason"
```

Comment:
It seems like most files don't have quotes around extension url values, for consistency, could we remove these and the others in the file?

Reply:
done